test/ibmq/test_ibmq_factory.py

```diff
@@ -269,7 +268,7 @@ def test_load_account_saved_provider_invalid_format(self, qe_token, qe_url):
                 with no_file('Qconfig.py'), custom_qiskitrc() as temp_qiskitrc, \
                         no_envs(CREDENTIAL_ENV_VARS):
                     # Save the account.
-                    self.factory.save_account(token=qe_token, url=qe_url)
+                    self.factory.save_account(token=self.token, url=AUTH_URL)
```

Comment:
The test does not do validation on the API, so it doesn't require the actual API token.